File: TechBlazers-Part1/currency.py
conversion_choice = int(input("Enter 0 (USD to naira) or 1 (naira to USD): "))
amount = int(input("Enter amount to be converted: "))

if conversion_choice == 0:
	print(f"Amount in naira: #{amount * 1500}")
elif conversion_choice == 1:
	print(f"Amount in Dollars: ${amount // 1500}")
else:
	print("Enter a valid input!!!")

File: TechBlazers-Part1/Currency.java
import java.util.Scanner;

public class Currency{

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter 0 (USD to naira) or 1 (naira to USD): ");
        int conversionChoice = input.nextInt();

        System.out.print("Enter amount to be converted: ");
        int amount = input.nextInt();

        if (conversionChoice == 0) {
            System.out.printf("Amount in naira: #%d\n", amount * 1500);
        } else if (conversionChoice == 1) {
            System.out.printf("Amount in Dollars: $%d\n", amount / 1500);
        } else {
            System.out.println("Enter a valid input!!!");
        }
    }
}